.lib-dist/README.md

```diff
@@ -0,0 +1,3 @@
+**NOTE**: This folder contains the compiled output of the `lib/` directory.
+You should be editting the `.ts` files there and then running `npm run build` or
```

Comment:
```suggestion
You should be editing the `.ts` files there and then running `npm run build` or
```

Reply:
This was in .sim-dist/README.md as well. Fixed in both places.